### PR TITLE
Mepts 1092

### PR DESCRIPTION
**Pull request by ningosi**
Fix column G and H

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWhoPickeupArvDuringPeriodDataDefinitionQueries.java [hunk 1]
```diff
@@ -80,109 +80,192 @@ public DataDefinition getLastStateOrDateFromARTProgramOrFichaResumoOrClinica(boo
...
+              + " WHERE  ps.state IN( ${state_6}, ${7}, ${8}, ${9}, ${10}, ${29} ) "
+              + " AND pg.location_id = :location "
+              + " AND ps.start_date <= :endDate "
+              + " AND p.voided = 0 "
+              + " AND pg.voided = 0 "
+              + " AND pr.retired = 0 "
+              + " AND ps.voided = 0) summary "
+              + " GROUP  BY patient_id) max_date_tbl "
+              + " ON s.patient_id = max_date_tbl.patient_id "
+              + " WHERE  s.state_date = max_date_tbl.state_date)fin ";
```
Comment (eliazmuchave) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWhoPickeupArvDuringPeriodDataDefinitionQueries.java]: Should format the query to improve the readability

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWhoPickeupArvDuringPeriodDataDefinitionQueries.java [hunk 2]
```diff
@@ -80,109 +80,192 @@ public DataDefinition getLastStateOrDateFromARTProgramOrFichaResumoOrClinica(boo
...
+              + " INNER JOIN patient_state ps "
+              + " ON pg.patient_program_id = ps.patient_program_id "
+              + " WHERE  ps.state IN( ${state_6}, ${7}, ${8}, ${9}, ${10}, ${29} ) "
+              + " AND pg.location_id = :location "
+              + " AND ps.start_date <= :endDate "
+              + " AND p.voided = 0 "
+              + " AND pg.voided = 0 "
+              + " AND pr.retired = 0 "
+              + " AND ps.voided = 0) summary "
+              + " GROUP  BY patient_id ";
```
Comment (eliazmuchave) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWhoPickeupArvDuringPeriodDataDefinitionQueries.java]: Should format the query to improve the readability

**Review by eliazmuchave — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWhoPickeupArvDuringPeriodDataDefinitionQueries.java (hunk 1 above)
Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWhoPickeupArvDuringPeriodDataDefinitionQueries.java]: This query is no longer on the code, it was re-implemented on the last fix

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWhoPickeupArvDuringPeriodDataDefinitionQueries.java (hunk 2 above)
Comment (helderjosue) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsWhoPickeupArvDuringPeriodDataDefinitionQueries.java]: This query is no longer on the code, it was re-implemented on the last fix

**Review by eliazmuchave — APPROVED** [review]
Looks good

**Pull request merged**